syntax/glsl.vim

```diff
@@ -26,15 +26,15 @@ syn region  glslPreProc         start="^\s*#\s*\(error\|pragma\|extension\|versi
 syn keyword glslBoolean true false
 
 " Integer Numbers
-syn match glslDecimalInt display "\(0\|[1-9]\d*\)"
-syn match glslOctalInt   display "0\o\+"
-syn match glslHexInt     display "0[xX]\x\+"
+syn match glslDecimalInt display "\(0\|[1-9]\d*\)[uUfF]\?"
+syn match glslOctalInt   display "0\o\+[uUfF]\?"
+syn match glslHexInt     display "0[xX]\x\+[uUfF]\?"
```

Comment:
Hello, and thanks for the review.
I think I misunderstood the Kronos wiki when they say ["To create a float literal from an integer value, use the suffix f or F as in C/C++"](https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)#Literals). Afaik something like `0f` is not valid C/C++ as well, and in C/C++ the `f` suffix on a floating point literal like `0.0f` makes it `float` instead of `double`. But if I understand well, floating literals in GLSL should already be `float` and the thing did not make much sense to me, so I implicitly assumed it was referred to integer literals.
Now I see on my machine the `f` suffix is valid on `float` literals (whatever it does), but I am not sure whether it is something standard or specific to mesa.